posthog/async_migrations/setup.py

```diff
@@ -82,6 +75,20 @@ def setup_async_migrations(ignore_posthog_version: bool = False):
         kickstart_migration_if_possible(first_migration, applied_migrations)
 
 
+def setup_model(migration_name: str, migration: AsyncMigrationDefinition) -> Optional[AsyncMigration]:
+    if migration.is_shown():
+        sm = AsyncMigration.objects.get_or_create(name=migration_name)[0]
+
+        sm.description = migration.description
```

Comment:
probably don't need to overoptimize so :thisisfine: but we will be doing some unnecessary work for each (shown) migration any time an async migration setting changes

Reply:
Note the code is exactly the same as we had previously - just abstracted the two callsites

Reply:
Oh nevermind. Yeah, we are but this should be fine